docs/general/administration/installing.md

```diff
@@ -291,6 +291,33 @@ On your Cloudron you can install Jellyfin with a few clicks via the [app library
 The source code for the package can be found [here](https://git.cloudron.io/cloudron/jellyfin-app).
 Any issues or feature requests related to deployment on Cloudron should be filed there.
 
+### Multipass
+
+[Multipass](https://multipass.run) can launch and run virtual machines and configure them with cloud-init like a public cloud. Get an instant Ubuntu VM with a single command. Prototype your cloud launches locally for free.
+
+1. Once multipass is installed, create a new instance with jellyfin image
+
+    ```sh
+    multipass launch jellyfin --name jellyfin --cpus 1 --disk 40G --mem 4G
+    ```
+
+2. Check that jellyfin is running in your instance:
+
+    ```sh
+    multipass exec jellyfin -- sudo service jellyfin status
+    # ● jellyfin.service - Jellyfin Media Server
+    #      Loaded: loaded (/lib/systemd/system/jellyfin.service; enabled; vendor pres>
+    #     Drop-In: /etc/systemd/system/jellyfin.service.d
+    #              └─jellyfin.service.conf
+    #      Active: active (running) since Wed 2022-05-04 11:55:38 CEST; 1min 37s ago
+    ```
+
+3. Further information:
+
+- Since this [pull request](https://github.com/canonical/multipass-blueprints/pull/18) is merged in blueprints, jellyfin is officially integrated in multipass.
```

Comment:
This line feels redundant, with the launch command we already imply jellyfin is available by default.

Reply:
Removed